### PR TITLE
fix URLs

### PR DESCRIPTION
**Pull request by sduthil**
Why:

* `make test` found errors in linked URLs

### CODE REVIEW

File: content/uc-doc/upgrade/index.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
 ---
 title: Upgrading
 ---
-
-- [Upgrading Procedure](/uc-doc/upgrade/introduction)
-- [Upgrade notes](/uc-doc/upgrade/upgrade_notes)
-- [Upgrade to a specific version of Wazo](/uc-doc/upgrade/upgrade_specific_version)
```
Comment (chartrandf) [inline content/uc-doc/upgrade/index.md:7]: Theses links, do not looks like `404`

File: tests/src/404.js [hunk 2]
```diff
@@ -52,20 +52,26 @@ const checkUrl = async (browser, url, fromUrl) => {
...
     });
 
     await browserPage.goto(url, { waitUntil: 'networkidle2' });
 
-    if (response && response.status() > 401) {
-      throw new Error(`status ${response.status()}`);
+    mainResponse = responses.find((response) => response.url() === url);
+    isResponseFailed = (response) => response && response.status() >= 400;
+    failedResponses = responses.filter((response) => isResponseFailed(response));
+    allResponsesOk = failedResponses.length === 0;
```
Comment (fblackburn1) [inline tests/src/404.js:72]: Not sure if @bogue will approve these old style variable declaration (without `let`/`const`) :stuck_out_tongue_closed_eyes:

File: content/uc-doc/upgrade/introduction.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ There are 2 options you can pass to wazo-upgrade:
 
 ### Upgrading from Wazo 18.03 only
 
-When upgrading from Wazo 18.03, you should follow the specific procedure for [Upgrading from Wazo 18.03](/uc-doc/upgrade/upgrade-from-wazo-18.03).
+When upgrading from Wazo 18.03, you should follow the specific procedure for [Upgrading from Wazo 18.03](/uc-doc/upgrade/upgrade_from_wazo_18.03).
```
Comment (chartrandf) [inline content/uc-doc/upgrade/introduction.md:50]: Looks like a `404` again, Gatbsy seems to render `/uc-doc/upgrade/upgrade_from_wazo_18`

File: content/uc-doc/upgrade/index.md (hunk 1 above)
Comment (sduthil) [reply content/uc-doc/upgrade/index.md:7]: No, by AFAICS, they are not visible anywhere, are they?

File: content/uc-doc/upgrade/introduction.md (hunk 3 above)
Comment (sduthil) [reply content/uc-doc/upgrade/introduction.md:50]: Thank you! Fixed in https://github.com/wazo-platform/wazo-platform.org/pull/365

File: content/uc-doc/upgrade/index.md (hunk 1 above)
Comment (chartrandf) [reply content/uc-doc/upgrade/index.md:7]: Yes, from here : https://wazo-platform.org/uc-doc/upgrade/ , but well you need to manipulate the URL to have access to it. Not that critical, it's okay 😄

**Pull request merged**